galaxy.yml

```diff
@@ -9,7 +9,7 @@ namespace: usegalaxy_eu
 name: handy
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 2.2.0
+version: 2.12.0
```

Comment:
If the collection must be compatible with semantic versioning I think looking at the other changes you must definitely increase the major version number (e.g. 3.0.0) because the changes will break any existing playbook.

Reply:
I'm also a fan of semantic versioning, but the new feature looks very convenient otherwise, thank you @sanjaysrikakulam :)

Reply:
Thanks for [56b2def](https://github.com/usegalaxy-eu/ansible-collection-handy/pull/13/commits/56b2def8f84ad16114604c3a3271e76df3d54ada)!